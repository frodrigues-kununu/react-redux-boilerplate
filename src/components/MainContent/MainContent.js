import React from 'react';

const MainContent = () => (
  <main className="page-content">
    main content here
  </main>
);


export default MainContent;
