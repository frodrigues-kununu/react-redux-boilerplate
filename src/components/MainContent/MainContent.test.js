import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';
import MainContent from './MainContent';

Enzyme.configure({ adapter: new Adapter() });

function setup() {
  const props = {};

  const enzymeWrapper = shallow(<MainContent {...props} />);

  return {
    props,
    enzymeWrapper,
  };
}

//test wrapper and props
const { enzymeWrapper, props } = setup();

describe('<MainContent />', () => {
  //snapshot testing
  test('should render correctly', () => {

    expect(toJSON(enzymeWrapper)).toMatchSnapshot();
  });
});
