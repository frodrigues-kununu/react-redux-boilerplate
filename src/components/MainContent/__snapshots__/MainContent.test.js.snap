// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContent /> should render correctly 1`] = `
<main
  className="page-content"
>
  main content here
</main>
`;
